.travis.yml

```diff
@@ -18,9 +18,7 @@ services:
 before_install:
   - git clone https://github.com/oceanprotocol/barge
   - cd barge
-  - export PROVIDER_VERSION=ptalgs
```

Comment:
This change doesn't seem to "strip JSON spaces", and there is no context to what it's for. 

Could you either update the PR description (and ensuing commit message) to give context, or put this in a different PR?